scipy/stats/_morestats.py

```diff
@@ -1133,7 +1133,9 @@ def _boxcox_inv_lmbda(x, y):
     return np.real(-num / np.log(x) - 1 / y)
 
 
-def boxcox_normmax(x, brack=None, method='pearsonr', optimizer=None):
+def boxcox_normmax(
+    x, brack=None, method='pearsonr', optimizer=None, *, ymax=None
```

Comment:
Consider creating an `object` other than `None` to represent the default. `ymax=None` in the signature suggests that there is no upper limit (infinity), but `ymax=BIG_FLOAT` would be consistent with the true behavior.

Reply:
Do I need to add a function called `BIG_FLOAT` to do this job?
```python
dtype = x.dtype if np.issubdtype(x.dtype, np.floating) else np.float64
ymax = np.finfo(dtype).max
```

Reply:
No. I'll make a specific code suggestion later.

Reply:
See xuefeng-xu/scipy#1